src/sync.rs

```diff
@@ -800,12 +802,22 @@ impl<T: Copy> LockFreeFrozenVec<T> {
     }
 
     pub fn is_empty(&self) -> bool {
-        self.len.load(Ordering::Relaxed) == 0
+        self.len() == 0
```

Comment:
thought: this changes the is_empty check to being Acquire, not Relaxed. Which is fine, I guess, I don't see either as being more correct here.